src/shorthands/JCircle.jl

```diff
@@ -39,3 +39,58 @@ JCircle(p1::Point, p2::Point; kwargs...) =
     JCircle(midpoint(p1, p2), distance(p1, p2) / 2; kwargs...)
 
 JCircle(radius::Real; kwargs...) = JCircle(O, radius; kwargs...)
+
+"""
+TODO: Add Documentation!
```

Comment:
currently I have no idea what `:inset` and `:clip` mean to be honest :smile: 
Seems like `:clip` positions the image somewhere else but I don't really have the option to move the image or have I?
Also for me a `scale_factor` is a number and not a symbol. Maybe `scale_type` or something but I would first need to know what they do to give my comment on that :smile: